storage/innobase/include/ut0ut.h

```diff
@@ -88,15 +88,25 @@ struct ut_when_dtor {
    the YieldProcessor macro defined in WinNT.h. It is a CPU architecture-
    independent way by using YieldProcessor. */
 #  define UT_RELAX_CPU() YieldProcessor()
-# elif defined(HAVE_ATOMIC_BUILTINS)
```

Comment:
@grooverdan I'm still evaluating possible options for ARM. I'll submit separate PRs for MySQL and likely MariaDB.